index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header class="h1-style">
+        <p>Hello Microverse</p>
+    </header>
```

Comment:
- [ ] In the project requirements, the index.html file includes a header with the text "Hello Microverse!" to use the `h1 tag`. So it's better to use the HTML `h1 tag` instead of `p tag.`